src/browser/main.coffee

```diff
@@ -118,7 +127,7 @@ parseCommandLine = ->
   options.boolean('portable').describe('portable', 'Set portable mode. Copies the ~/.atom folder to be a sibling of the installed Atom location if a .atom folder is not already there.')
   options.alias('t', 'test').boolean('t').describe('t', 'Run the specified specs and exit with error code on failures.')
   options.string('timeout').describe('timeout', 'When in test mode, waits until the specified time (in minutes) and kills the process (exit code: 130).')
-  options.alias('v', 'version').boolean('v').describe('v', 'Print the version.')
+  options.alias('v', 'version').boolean('v').describe('v', 'Print the version information.')
```

Comment:
I thought about that at first. I want to make it drop-dead simple though. I also want Atom to always display the same version information ... whether at the command line or in the About view. This way if Atom won't launch (or only shows the white screen), you can still get to the version info from the command line.